### PR TITLE
enhance logs during ab install

### PR DESCRIPTION
**Pull request by nh758**
## Release Notes

- consistent, user friendly, and simple log outputs during install
- add a `--verbose` flag to output more detailed logs
- add a progress bar to the tar extract
- add a developer instructions after install 
 

## Test Status
E2E tests confirm install works

## Screenshots
<details><summary>Log Output</summary>
<p>

image

</p>
</details>
<details><summary>Verbose Log Output</summary>
<p>

image

</p>
</details>

### CODE REVIEW

File: lib/tasks/configTenantAdmin.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const inquirer = require("inquirer");
 const { nanoid } = require("nanoid");
 const path = require("path");
 const shell = require("shelljs");
-const utils = require(path.join(__dirname, "..", "utils", "utils"));
+const utils = require("../utils/utils");
```
Comment (Hiro-Nakamura) [inline lib/tasks/configTenantAdmin.js:14]: in the past, we used the `path.join()` command to ensure proper platform compatibility with referencing files.

Is this no longer a concern for cross platform compatibility?

Comment (Hiro-Nakamura) [reply lib/tasks/configTenantAdmin.js:14]: (and by platform, I mean `windows` vs `macOS` vs `linux` etc... )

File: lib/utils/dockerStackWatch.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ module.exports = function (
          [
             (done) => {
                // be sure there are no other ab stacks running:
-               console.log(`... clearing out existing ${stack} stacks`);
+               console.log(`    ... clearing out existing ${stack} stacks`);
```
Comment (Hiro-Nakamura) [inline lib/utils/dockerStackWatch.js:74]: would it be worth putting your `log()` and `warn()` shortcuts in here too?

**Review by Hiro-Nakamura — APPROVED** [review]
Overall this looks good.

If you don't have any issues with the questions then just commit!

File: lib/tasks/configTenantAdmin.js (hunk 1 above)
Comment (nh758) [reply lib/tasks/configTenantAdmin.js:14]: That may have been needed in older versions of node, but it shouldn't be necessary anymore.

**Pull request merged**